src/main/java/core/basesyntax/storageq/StorageInformation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.storageq;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageInformation {
+    private static Map<String, Integer> shopReport = new HashMap<>();
+
+    public static Map<String, Integer> getShopReport() {
+        return shopReport;
+    }
+
+    public static void putShopReport(String fruit, Integer value) {
+        shopReport.put(fruit, value);
+    }
```

Comment:
You should **OR** always use your map directly (you can make it public in this case) **OR** make it private, remove getter, and have different public methods for working with storage.